src/statusTile/updateStatusTileScope.js

```diff
@@ -3,14 +3,14 @@ const { getCurrentScope } = require('../editorInterface');
 const { getAllScopes } = require('../atomInterface');
 
 const updateStatusTileScope = (element: HTMLElement, editor: TextEditor) => {
-  if (element) {
-    // The editor can be undefined if there is no active editor (e.g. closed all tabs).
-    // eslint-disable-next-line no-param-reassign
-    element.dataset.prettierMatchScope = editor !== undefined &&
-      getAllScopes().includes(getCurrentScope(editor))
-      ? 'true'
-      : 'false';
-  }
+  if (!element) return;
+
+  // The editor can be undefined if there is no active editor (e.g. closed all tabs).
+  // eslint-disable-next-line no-param-reassign
+  element.dataset.prettierMatchScope = editor !== undefined &&
```

Comment:
We set `display: none` in the `statusTile` stylesheet if this attribute is set to `"false"`, so it would not work if we returned `undefined`.
I'm not sure I understood correctly though, does that answer your question?